pkgs/development/python-modules/manimpango/pangocairo.patch

```diff
@@ -0,0 +1,21 @@
+diff --git a/setup.py b/setup.py
```

Comment:
Perhaps this could be done using `substituteInPlace`, to avoid breakage in future updates?

Reply:
done!